src/java/io/bazel/rulesscala/scalac/ScalacProcessor.java

```diff
@@ -270,18 +270,27 @@ private static void compileJavaSources(CompileOptions ops, String[] javaSources,
     commandParts.add("@" + normalizeSlash(argsFile.toFile().getAbsolutePath()));
     try {
       Process iostat = new ProcessBuilder(commandParts)
-        .inheritIO()
         .start();
+      String error = getInputAsString(iostat.getErrorStream());
+      String output = getInputAsString(iostat.getInputStream());
       int exitCode = iostat.waitFor();
       if(exitCode != 0) {
-        throw new RuntimeException("javac process failed!");
+        throw new RuntimeException("javac process failed! "+ error + " " + output);
+      } else {
+        System.out.println("Javac stdout: " + output);
```

Comment:
do we need this in the general case? I'm afraid it would add a bunch of noise to many builds.